package.json

```diff
@@ -44,8 +44,9 @@
   ],
   "scripts": {
     "build": "node scripts/inject-version.js && tsc --emitDeclarationOnly",
-    "docs:start": "astro dev --root ./docs",
-    "docs:build": "astro check  --root ./docs && astro build --root ./docs",
+    "docs:token-build": "npx tsx docs/theme/build.ts",
```

Comment:
`tsx` should also be installed as a devDependency imo, I think it works for you locally because you've probably got it installed globally?